docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/sitewideSearch/SitewideSearchApp.js

```diff
@@ -4,7 +4,7 @@
  */
 
 import initializeSitewideSearchApp from "Libraries/sitewideSearch/sitewideSearchApp";
-import "./SitewideSearchApp.scss";
+// import "./SitewideSearchApp.scss";
```

Comment:
Is this meant to be removed? Or is this a "Trying something and forgot to uncomment it?"

Reply:
same as previous